ast/visit.go

```diff
@@ -592,8 +607,15 @@ func (c *ClassLiteral) VisitWith(v Visitor) {
 }
 
 func (c *ClassLiteral) VisitChildrenWith(v Visitor) {
-	c.Name.VisitWith(v)
-	c.SuperClass.VisitWith(v)
+	if c.Name != nil {
+		c.Name.VisitWith(v)
+	}
+	if c.SuperClass != nil {
+		(*c.SuperClass).VisitWith(v)
+	}
+	for _, element := range c.Body {
+		element.(VisitableNode).VisitWith(v)
```

Comment:
Should just make the `ClassElement` interface embed `VisitableNode`

Reply:
Resolved in #8